src/main/java/lotto/domain/NumberGenerator.java

```diff
@@ -0,0 +1,29 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class NumberGenerator {
+    private static final int MIN_LOTTO_NUMBER = 1;
+    private static final int MAX_LOTTO_NUMBER = 45;
+
+    private static final List<Integer> numberRange = Stream.iterate(MIN_LOTTO_NUMBER, n -> n + 1)
+            .limit(MAX_LOTTO_NUMBER)
+            .collect(Collectors.toList());
```

Comment:
```suggestion
    private static final List<Integer> numbers = IntStream
            .rangeClosed(MIN_LOTTO_NUMBER, MAX_LOTTO_NUMBER)
            .mapToObj(Integer::new)
            .collect(toList());
```
iterate하면서 연산할 필요 없이 IntStream 을 활용하여 범위내의 값을 인스턴스화 할 수 있습니다.

Reply:
오 새로운 지식 감사합니다! :)
반영해서 수정해볼게요!